aws/ci_build

```diff
@@ -144,11 +144,31 @@ Showing last #{CHAT_LOG_TAIL_LENGTH} lines of build log:
       ChatClient.message 'server operations', message, color: 'red', notify: 1
       ChatClient.message 'server operations', commit_url, color: 'gray', message_format: 'text'
     end
+
+    emit_deploy_metric(rack_env.to_s.titleize, status == 0)
```

Comment:
Throughout our codebase, we tend not to titleize our environment names. Why do so here?